LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -1218,6 +1663,16 @@ private void HandleClientStatusReport(ClientStatusReport msg)
 
         private void HandleSubscribeToCustomGamesRequest(SubscribeToCustomGamesRequest request)
         {
+            List<LobbyGameInfo> customGameInfos = new List<LobbyGameInfo>();
```

Comment:
I guess there is no update button, the server is supposed to send updates periodically?

We can probably leave it for later.

Reply:
Could be done , there is a UnsubscribeFromCustomGamesRequest to